WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -73,9 +73,8 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 			static bool? MaybeCoinjoining(StatusChangedEventArgs args) =>
 				args switch
 				{
-					StartedEventArgs _ => true,
-					StoppedEventArgs _ => false,
-					CompletedEventArgs => false,
```

Comment:
What was this and why was it removed?

Reply:
I asked this when I naively thought I can review this PR within an hour, but I failed to understand 90% of what's going on here, so nitpicking to understand this won't make a difference.

Reply:
This sets spectrumcontrol if it is turned on off or critical. Just controls the animation.